src/porepy/models/compositional_flow.py

```diff
@@ -0,0 +1,2070 @@
+  for overall fractions and active tracer fractions, if any. Other fractions are for now
+  not covered since they are considered secondary and their initialization can be done
+  based on the primary fractions.
+- :class:`InitialConditionsPhaseProperties`: An initialization routine for models where
+  phase properties are represented by :class:`~porepy.numerics.ad.surrogate_operator.
+  SurrogateFactory`. Their initialization is dependent on variable values, and is
+  automatized to happen after the variables are initialized.
+- :class:`InitialConditionsFlowAndTransport`: A collection of above initialization
+  routines, and the IC mixins for mass & energy, including an independent enthalpy
+  variable.
+
+Following BC mixins are available:
+
+- :class:`BoundaryConditionsCF`: The analogy to
+  :class:`InitialConditionsCF` but for BC.
+- :class:'BoundaryConditionsPhaseProperties': The analogy to
+  :class:`InitialConditionsPhaseProperties` but for BC. Their update can be automatized
+  to happen after values for variables are set on the boundary.
+- :class:`BoundaryConditionsFractionalFlow`: An alternative to
+  :class:`BoundaryConditionsPhaseProperties` for the fractional flow setting, where
+  various non-linear terms in fluxes can be given explicitly on the boundary,
+  instead of providing variable values which in return compute phase properties
+  appearing in those expressions.
+- :class:`BoundaryConditionsFlowAndTransport`: A collection of BC update routines for
+  primary variables and phase properties as surrogate factories, including those from
+  mass & energy, enthalpy and fractions.
+- :class:`BoundaryConditionsFractionalFlowAndTransport`: The alternative to
+  :class:`BoundaryConditionsFlowAndTransport` for the fractional flow setting with
+  explicit values for non-linear weights in fluxes.
+
+The :class:`SolutionStrategyCF` handles the general Cf model, with or without fractional
+flow simulation, and provides means to re-discretize the MPFA in the diffusive setting,
+and to eliminate local, secondary equations via Schur-complement. Those equations are
+required in any case to close the general multi-phase setting.
+
+The two setups, :class:`ModelSetupCF` and :class:`ModelSetupCFF` are in principle
+complete set-ups for non-isothermal, compositional flow. The steps required by users to
+close the setups are:
+
+1. Define a fluid with all its phases and components.
+2. Close the system with local equations for dangling, fractional variables. These can
+   either be :class:`~porepy.models.abstract_equations.LocalElimination` using some
+   map or third-party correlations, or a closure in form of a local equilibrium system.
+
+"""
+
+from __future__ import annotations
+
+import logging
+import time
+from functools import partial
+from typing import Callable, Optional, Sequence, cast
+
+import numpy as np
+
+import porepy as pp
+import porepy.compositional as ppc
```

Comment:
This breaks with the import style for subpackages. Can we rather come up with a better suggestion of the type pp.comp. or similar? @IvarStefansson: Ideas?

Reply:
My first suggestion would be `from porepy import compositional`

Reply:
Changed according to Ivar's suggestion.